mock/mockimpl/svcimpls/mgmtimplcollections.go

```diff
@@ -291,57 +291,63 @@ func (x *mgmtImpl) handleGetAllScopes(source mock.MgmtService, req *mock.HTTPReq
 	}
 
 	manifest := bucket.CollectionManifest()
-	uid, scopes := manifest.GetAllScopes()
+	uid, scopes := manifest.GetManifest()
 
+	jsonMani := buildJSONManifest(uid, scopes)
+
+	b, err := json.Marshal(jsonMani)
+	if err != nil {
+		return &mock.HTTPResponse{
+			StatusCode: 500,
+			Body:       bytes.NewReader([]byte(err.Error())),
+		}
+	}
+
+	return &mock.HTTPResponse{
+		StatusCode: 200,
+		Body:       bytes.NewReader(b),
+	}
+}
+
+func buildJSONManifest(uid uint64, scopes []mock.CollectionManifestScope) jsonManifest {
```

Comment:
At some point this will likely need to hang off `kvImplCrud` to have access to the version we are emulating, but this should be fine for now.